topics/metagenomics/tutorials/foodborne-pathogen-detection-nanopore-sequencing-data/tutorial.md

```diff
@@ -0,0 +1,653 @@
+In order to view the taxonomy profiling produced by **Kraken2** tool there are a lot of tools to be used afterwards such as **Krona pie chart**, however the species classified are a lot to be shown by this tool. For that reason, we have chosen the **Phinch visualization** interactive tool as it contains multiple visualization plots, it's interactive you can choose between different parameters, you can visualize each taxonomic level on its own, you can have your metadata of the samples represented along with the taxonomic visualization, download all plots for publications and a lot of other benefits. For later, you can check out [__Pavian__](https://academic.oup.com/bioinformatics/article/36/4/1303/5573755) tool as well it can replace **Phinch visualization** with similar outputs.
+
+**Phinch visualization** needs a **Biom** file format as an input for that we needed the **Kraken-Biom** tool to convert the **Kraken2** tabular output into a Biom file. 
+
+Now lets try the **Phinch visulization** tool running for `Barcode11` history
+
+> <hands-on-title>Explore data interactively</hands-on-title>
+>
+> 1. Open interactive tool
+>
+>    {% snippet faqs/galaxy/interactive_tools_open.md tool="Phinch visualization" %}
+>
+> 2. Click on **Proceed to Gallery** button on the top right of the opened webpage to see all the plots
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. What is the most found species?
+> 2. What is the second most found species?
+> 3. What's your favorite visualization plot?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Salmonella enterica with 17,309 sequences
+> > 2. Pseudomonas lundensis with 13,227 sequences
+> > 3. All of them are good visualization to the data but for us to answer these questions, we used **Taxonomy Bar chart**
+> >
+> >    ![Taxonomy Bar Chart](./images/phinch_taxonomy_bar_chart.png)
+> >
+> {: .solution}
+{: .question}
+ 
+# Gene based pathogenic identification
+In this section we will analyze the sequences in our samples and identify genes that have a [Virulence Factor (VF)](https://www.sciencedirect.com/topics/immunology-and-microbiology/virulence-factor), which will then define a pathogen and its severity level. We will also identify [Antimicrobial Resistance genes (AMR)](https://www.sciencedirect.com/topics/engineering/antibiotic-resistance-genes), which their presence will destroy the antibiotic that will lead to not altering the target site and spread throughput the pathogenic bacteria decreasing the overall fitness of the host. Using [Multilocus Sequence Typing (MLST)](https://pubmlst.org/) database we will also determine the strain of the bacteria we are testing for pathogenicity. Never the less, we will be creating our **FASTA** and **Tabular** files that we will used later on to track and visualize our pathogenic identification throughout all samples, which is done in the **Pathogen Tracking among all samples** section and sub-workflow coming later in this tutorial.
```

Comment:
I think this paragraph is not very accurate; genes "cannot have a virulence factor", as genes encode the virulence factors (virulence factors are gene products, usually proteins, involved in pathogenicity). Also Resistance genes not allways act by destroying the antibiotics: The three fundamental mechanisms of antimicrobial resistance are enzymatic degradation of antibacterial drugs,  alteration of bacterial proteins that are antimicrobial targets, and changes in membrane permeability to antibiotics (source: https://pubmed.ncbi.nlm.nih.gov/2025137/#:~:text=The%20three%20fundamental%20mechanisms%20of,in%20membrane%20permeability%20to%20antibiotics)